src/unity/schemas/IdentityOrganization.yml

```diff
@@ -0,0 +1,15 @@
+properties:
+  idpeId:
+    type: string
+    description: idpe ID of the org
+  name:
+    type: string
+    description: name of the org
+  clusterHost:
+    type: string
+    description: the url of the current cluster
+required:
+  - id
```

Comment:
This line should no longer be here, since there isn't an `id` field any more. Unless we rename `idpeId` to `id`, then we'd want to delete `idpeId` from the line below.